src/main/java/com/google/sps/data/ComputeAPIClientInterface.java

```diff
@@ -0,0 +1,39 @@
+package com.google.sps.data;
+
+import com.google.api.services.compute.model.Operation;
+import java.io.IOException;
+import java.util.Optional;
+
+/** Interface for the ComputerAPIClient class. */
+public interface ComputeAPIClientInterface {
+  /** 
+   * Creates a new Instance.
+   * @param {String} instanceName - the name of the new Instance.
+   * @param {String} vncServerPassword - the password needed to access 
+   *    to vnc server.
+   */
+  void createInstance(String instanceName, String vncServerPassword) 
```

Comment:
I noticed that this method throws IOException but the others throw Exception. Although honestly, it's not best practice, lets make this throw Exception as well, for consistency.